backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5709670_sys_remove_bpartner_document_copies_2.sql

```diff
@@ -0,0 +1,6 @@
+-- Column: C_BPartner.DocumentCopies
+-- Column: C_BPartner.DocumentCopies
+-- 2023-11-06T16:58:59.167Z
+UPDATE AD_Column SET IsActive='N',Updated=TO_TIMESTAMP('2023-11-06 17:58:59','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Column_ID=3086
```

Comment:
loud thinking (not sure): 
1. don't we need to migrate C_BPartner.DocumentCopies to something else
2. why not backing up current c_bpartner table and then drop the DocumentCopies column?

Reply:
1. We can add a bp_print_fornat with only copies, to archive the same result as bpartner one before
2. It was the faster way without cleaning constraints to this columns and less diskspace intensive

Reply:
dropped in `5709710_sys_remove_bpartner_document_copies.sql`